OpenRiaServices.NuGet/OpenSilver/OpenRiaServices.OpenSilver.Client.Core.nuspec

```
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
   <metadata>
-    <id>OpenSilver.OpenRiaServices.Client.Core.4.6</id>
+    <id>OpenSilver.OpenRiaServices.Client.Core</id>
```

We need to keep 4.6 at the end as we are maintaining 2 OpenRia packages. One based on version 5 of OpenRia (the id of the NuGet package is OpenSilver.OpenRiaServices.Client.Core) and one based on the version 4.6 for Silverlight compatibility (NuGet package is OpenSilver.OpenRiaServices.Client.Core.4.6)
```suggestion
    <id>OpenSilver.OpenRiaServices.Client.Core.4.6</id>
```